IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/NewsCollectionViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  NewsCollectionViewCell.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+import SDWebImage
+
+final class NewsCollectionViewCell: UICollectionViewCell {
+
+    static let identifier = "NewsCollectionViewCell"
+    static let Height = 80.0
+    
+    @IBOutlet weak var imgDetail: UIImageView!
+    @IBOutlet weak var lblDate: UILabel!
+    @IBOutlet weak var lblDescription: UILabel!
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblAuthor: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.contentView.frame = CGRect(x: self.contentView.frame.origin.x, y: self.contentView.frame.origin.y, width:UIScreen.main.bounds.width , height: CGFloat(NewsCollectionViewCell.Height))
+    }
+
+    func setup(viewModel: NewsArticleViewModel) {
```

Comment:
> This is the same case that the previous comment @bsztamfater-ms & @mvazquez-ms

My comment was about defining properties as observable or not, that's related but no the same as binding.

To be consistent through out the app, I think we should always communicate views and data in the same way.  Here you will have one-time binding (evaluated only when the view is created, and they don't require change notifications).

We'll definitely use binding in many places, so to keep consistency, it's better to use it everywhere. And chances are most ViewModels will have notifiable properties.. a NewsArticle can have a `comments` or `likes` indicator that should be notifiable to be update in real time.